mfr/extensions/codepygments/render.py

```diff
@@ -20,6 +21,9 @@ class CodePygmentsRenderer(extension.BaseRenderer):
         ]).get_template('viewer.mako')
 
     def render(self):
+        file_size = os.path.getsize(self.file_path)
+        if file_size > 65536:
```

Comment:
Could this be made configurable, similarly to how `tabular/settings.py` are set?